statics/src/test/java/integration/proxy/MultipleDownloadsTest.java

```diff
@@ -0,0 +1,27 @@
+package integration.proxy;
+
+import com.codeborne.selenide.files.FileFilters;
+import integration.IntegrationTest;
+import org.junit.jupiter.api.*;
+
+import java.io.*;
+
+import static com.codeborne.selenide.DownloadOptions.using;
+import static com.codeborne.selenide.FileDownloadMode.PROXY;
+import static com.codeborne.selenide.Selenide.*;
+
+public class MultipleDownloadsTest extends IntegrationTest {
+  @BeforeEach
+  void setUp() {
+    closeWebDriver();
```

Comment:
It's not needed to call `closeWebDriver();`: method `useProxy(true);` does it automatically if needed.

Reply:
fixed